dice.js

```diff
@@ -0,0 +1,26 @@
+const random = () => {
+  return Math.round(Math.random() * 5 + 1);
```

Comment:
While this code with give you a random number between `1` and `6` the chance of getting `1` or `6` is half likely than the other numbers.  

Use this to get a number between `1` and `6 ` ```  Math.floor(Math.random() * 6) + 1 ```
- ```Math.random``` will give random number between `0` and `0.99... `, it will never give `1`.
- so here ```Math.random() * 6```  will give number between `0` and `5.99...` 
- `Math.floor` will give you integer that equal or less than the given number so `Math.floor(2.987)` will return `2`.
- using ```Math.floor()``` will result to an integer between `0` and `5` and adding ```+1``` you will  end up  with an integer between `1` and `6`.

_if you have any question about this feel free to ask me_